src/current/v23.1/essential-metrics.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Essential Metrics to Monitor
```

Comment:
The Essential Metrics list was curated for Self-hosted and Dedicated. Some metrics are not relevant for serverless. Should we call this out?

Perhaps "Essential Metrics to Monitor" shouldn't be the first doc in this section. Did we consider having it under "Third-Party Monitoring Integrations"?

Reply:
With the new left nav structure, Serverless is not an issue.